# Comparative study related to SARS-Cov-2 (COVID-19) and its variant Omicron.
---
### Table of Contents:-

- [Description](#Description)
- [Demos](#Demos)
- [Tools](#Tools)
---

## Description
First Step, Download 10 SARS-Cov-2 sequences from the Gana country from GISAID website:-

- Construct a consensus sequence from these sequences. At each sequence location, the
nucleotide/amino acid of the consensus sequence will be the most dominant one across all
the sequences at that location.
- We will use this consensus sequence as a single representation for the reference sequences.
---
Second Step, Download 10 sequences for the SARS-Cov-2 Omicron variant:-

- We will call them the case sequences.
- Apply any multiple sequence alignment technique on these sequences.
---
Third Step, Ways of comparison:-

- The average percentage of the chemical constituents (C, G, T, and A) and the CG content, if
applicable, between the reference sequences and the case sequences.
- Extract the dissimilar regions/columns between the alignment of the case sequences and the
consensus sequence (the representative reference).
---

### Demos
---
-First Step:-
![https://github.com/Sandra-Essa/Comparative_Study_Related_To_SARS_COVID-19_And_Its_Variant_Omicron/blob/main/GIF/BioInformatics_GIF1.gif]
---
-Second Step:-
![https://github.com/Sandra-Essa/Comparative_Study_Related_To_SARS_COVID-19_And_Its_Variant_Omicron/blob/main/GIF/BioInformatics_GIF2.gif]
---
-Third Step:-
![https://github.com/Sandra-Essa/Comparative_Study_Related_To_SARS_COVID-19_And_Its_Variant_Omicron/blob/main/GIF/BioInformatics_GIF3.gif]
![https://github.com/Sandra-Essa/Comparative_Study_Related_To_SARS_COVID-19_And_Its_Variant_Omicron/blob/main/GIF/BioInformatics_GIF4.gif]
---

### Tools
----
- Python - Jupyter Notebook
- Fasta Website
- GISAID website
----
